enhancements/bootimages.md

```diff
@@ -0,0 +1,163 @@
+---
+title: bootimage-updates
+authors:
+  - "@cgwalters"
+reviewers:
+  - "@coreos-team"
+approvers:
+  - "@coreos-team"
+creation-date: 2020-02-04
+last-updated: 2020-02-024
+status: provisional
+---
+
+# Updating bootimages
+
+This proposes a path towards integrating the bootimage into the release image and become managed by the cluster.  This aids worker scaleup speed, helps in mirroring OpenShift into disconnected environments, and allows the CoreOS team to avoid supporting in-place updates from OpenShift 4.1 into the forseeable future.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+OpenShift will support fully clusters managing their own bootimages (in IaaS clouds), and come with tooling and documentation to aid operators on e.g. bare metal PXE to keep bootimages up date.
```

Comment:
nit: can you define IaaS somewhere in the doc?

Reply:
Seems simpler to drop the term, most people understand what "cloud" means here i think.